src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/addDriverToCar.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/add.jsp").forward(req, resp);
```
check all endpoint names. Don't use camelCase, better use sub packages and one word for the name (if it's impossible, use a hyphen). pay attention to the link to your PR https://github.com/mate-academy/jv-web-practice/pull/409/files (without camelCase)
<img width="509" alt="image" src="https://user-images.githubusercontent.com/96740815/203334258-48fb9a32-a53a-4181-83c4-6e5d88439b33.png">

Reply:
+ All paths & endpoints renemed in new commit.